winton_kafka_streams/processor/topology.py

```diff
@@ -31,46 +31,63 @@ def __repr__(self):
 
 
 class Topology:
-    def __init__(self, _nodes, _sources, _processors, _sinks, _state_stores):
-        self.nodes = _nodes
-        self.sources = _sources
-        self.processors = _processors
-        self.sinks = _sinks
-        self.state_stores = _state_stores
+    """
+    A realised instance of a toplogy
+
+    """
+    def __init__(self, sources, processors, sinks, state_stores):
+        self.nodes = {}
+        self.sources = {}
```

Comment:
Sources now kept in a dictionary to ensure we can associate a topic with the correct processor in stream task.